Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer'
+import CardContainer from './components/CardComponents/CardContainer'
 import './App.css';
 
 const App = () => {
+  let props = {
+    header: {
+      thumbnail: {
+        src: 'https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png',
+        alt: 'Lambda School',
```

Comment:
great job picking a good form of data type to put your data in, this makes it much easier to go in and grab what you want. I would not recommend using the props name as the wording however as that isn't really what this is and can be confusing or mess you up if mixed with actual props.